tests/SPR.tex

```diff
@@ -92,6 +92,7 @@ \subsection{OC instructions}
 \begin{itemize}
     \item Announce the placement of the robots
     \item choose the volunteers for the second part of the test, and clearly explain the procedure to them.
+    \item Announce where the robot should leave to after the test. 
```

Comment:
I find this unnecessary as it over-specifies the function of the Exit door in General Rules. In any case I would say:

```When the test is held outside the arena, announce the leaving [way]point```